pkg/controller/csi-driver-lvm/actuator.go

```diff
@@ -0,0 +1,807 @@
+		ReclaimPolicy:        &reclaimPolicy,
+		VolumeBindingMode:    &volumeBindingMode,
+		AllowVolumeExpansion: pointer.Pointer(true),
+		Parameters: map[string]string{
+			"type": "linear",
+		},
+	}
+
+	csiNodeDriverRegistrarImage, err := imagevector.ImageVector().FindImage("csi-node-driver-registrar")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find csi-node-driver-registrar image: %w", err)
+	}
+
+	livenessprobeImage, err := imagevector.ImageVector().FindImage("livenessprobe")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find livenessprobe image: %w", err)
+	}
+
+	csiDriverLvmImage, err := imagevector.ImageVector().FindImage("csi-driver-lvm")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find csi-driver-lvm image: %w", err)
+	}
+
+	csiDriverLvmProvisionerImage, err := imagevector.ImageVector().FindImage("csi-driver-lvm-provisioner")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find csi-driver-lvm-provisioner image: %w", err)
+	}
+
+	// var terminationPolicy corev1.TerminationMessagePolicy = corev1.TerminationMessageReadFile
+	var mountPropagation corev1.MountPropagationMode = corev1.MountPropagationBidirectional
+
+	var hostPathTypeCreate corev1.HostPathType = corev1.HostPathDirectoryOrCreate
+	var hostPathTypeDir corev1.HostPathType = corev1.HostPathDirectory
+
+	csidriverlvmDaemonSetPlugin := &appsv1.DaemonSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "csi-driver-lvm-plugin",
+			Namespace: namespace,
+		},
+		Spec: appsv1.DaemonSetSpec{
+			RevisionHistoryLimit: ptr.To(int32(10)),
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"app": "csi-driver-lvm-plugin",
+				},
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app": "csi-driver-lvm-plugin",
+					},
+				}, Spec: corev1.PodSpec{
+					ServiceAccountName: "csi-driver-lvm-plugin",
+					Tolerations:        []corev1.Toleration{},
+					NodeSelector:       map[string]string{},
+					Containers: []corev1.Container{
+						{
+							Name:            "csi-node-driver-registrar",
+							Image:           csiNodeDriverRegistrarImage.String(),
+							ImagePullPolicy: "IfNotPresent",
+							Args:            []string{"--v=5", "--csi-address=/csi/csi.sock", "--kubelet-registration-path=/var/lib/kubelet/plugins/csi-driver-lvm/csi.sock"},
+							SecurityContext: &corev1.SecurityContext{
+								ReadOnlyRootFilesystem: pointer.Pointer(false),
+								// Privileged:             pointer.Pointer(true),
+							},
+							Env: []corev1.EnvVar{
+								{
+									Name: "KUBE_NODE_NAME",
+									ValueFrom: &corev1.EnvVarSource{
+										FieldRef: &corev1.ObjectFieldSelector{
+											APIVersion: "v1",
+											FieldPath:  "spec.nodeName",
+										},
+									},
+								},
+							},
+							// TerminationMessagePath:   "/dev/termination-log",
+							// TerminationMessagePolicy: terminationPolicy,
```

Comment:
What was the issue with these properties?

Reply:
first I deployed it on the seed -> issues arised here
after fixing to seed there was no problem anymore